packages/appstore/src/components/account/account.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import WalletIcon from 'Assets/svgs/wallet';
+
+type TOptionsAccountprops = {
+    account_icon: string;
+    account_title?: string;
+    account_number?: string;
+    account_balance?: string;
+    account_button?: string;
+};
+
+const OptionsAccount = ({
+    account_icon,
+    account_title,
+    account_number,
+    account_balance,
+    account_button,
+}: TOptionsAccountprops) => {
+    return (
+        <div className='account__container'>
+            <div className='account__container--icon'>
+                <WalletIcon icon={account_icon} />
+            </div>
+            <div className='account__container--account-details-wrapper'>
+                <div className='account__container--account-details-wrapper--name-number'>
+                    <Text className='account__container--account-details-wrapper--name-number--name' weight='bold'>
+                        <Localize i18n_default_text={account_title} />
+                    </Text>
+                    <Text className='account__container--account-details-wrapper--name-number--number'>
+                        <Localize i18n_default_text={account_number} />
+                    </Text>
+                </div>
+
+                <Text className='account__container--account-details-wrapper--balance'>
+                    <Localize i18n_default_text={account_balance} />
```

Comment:
better to separate the balance and currency, like what we have in the Dtrader account info
![image](https://user-images.githubusercontent.com/82078941/188559292-4c059e2f-0a70-4f1d-bf31-9334d037ef66.png)